src/components/SessionTimer/SessionTimer.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../scss/variables.scss";
+
+.session-timer {
+  padding: $spacing-medium;
+  background-color: $color-primary-dark;
+  width: 140px;
+  color: $color-white;
+  font-size: $font-size-large;
+  border-radius: $border-radius;
+  text-align: center;
+  box-shadow: $box-shadow;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  position: relative;
```

Comment:
Im guessing all these styles are the same as the ones from `<LapCounter />`?

If they are should we create a parent `<SessionBanner />` component that renders **LapCounter** or **SessionTimer** depending on some prop

Reply:
Only difference is the addition of justify-content on this one.

Reply:
Added this here: [fbc0a95](https://github.com/KChadwick96/rFactor2-Overlay-v2/pull/1/commits/fbc0a9502f9af3863a975a6f46978b6c578e0914)